django/utils/timesince.py

```diff
@@ -6,6 +6,15 @@
 from django.utils.timezone import is_aware, utc
 from django.utils.translation import ugettext, ungettext_lazy
 
+chunks = (
```

Comment:
Can you make this upper case and give it a slightly more unambiguous name, e.g. `TIMESINCE_CHUNKS`